contracts/v1/assets/ContentAsset.sol

```diff
@@ -43,7 +43,7 @@ contract ContentAsset is Named, Versioned, HubDependent, Initializable {
     event AssetUpdatePaymentIncreased(address indexed assetContract, uint256 indexed tokenId, uint96 tokenAmount);
 
     string private constant _NAME = "ContentAsset";
-    string private constant _VERSION = "1.0.3";
+    string private constant _VERSION = "1.0.4";
```

Comment:
Why do we have version bump if nothing is changed here? Also should we change KA creation it to support adding KA to Paranets?

Reply:
I forgot to bump it when I did previous updates

Reply:
Changes you're referring to are added in the v2 folder